apps/frontend/static_src/scss/components/nav-buttons.scss

```
@@ -10,6 +10,7 @@
 
     &__link {
         display: flex;
+        justify-content: space-between;
```

I believe it would be better to use `flex-start` here as the default for a `__link`, and then apply `flex-end` to the second child:
```suggestion
        justify-content: flex-start;
        
        @include media-query(small) {
            &:nth-child(2) {
                justify-content: flex-end;
            }
        }
```